WebAPI.Tests/UnitTest1.cs

```diff
@@ -0,0 +1,89 @@
+using System.Text.Json.Serialization;
+using RestSharp;
+using WebAPI.Infrastructure;
+using WebAPI.Infrastructure.RequestsResponses;
+
+namespace WebAPI.Tests;
+
+public class Tests
+{
+    private RestApiClient _restClient;
+    private int _id;
+
+    [SetUp]
+    public async Task Setup()
+    {
+        _restClient = new RestApiClient();
+        await _restClient.Authenticate("admin", "password123");
+    }
+
+    [Test, Order(1)]
```

Comment:
Providing test order is a anti-pattern. Your tests should be independent. Create booking before each scenario to avoid test order.